### PR TITLE
Readme setup

### PR DESCRIPTION
**Pull request by HargovindArora**
#1  Please review it @haroon-sheikh

### CODE REVIEW

File: requirements/dev.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Django==1.11.3
```
Comment (haroon-sheikh) [inline requirements/dev.txt:1]: project is not running with 1.8 yet

File: README.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+1. Install [python] 2.x, git and [virtualenv] in your computer.
+
+2. Get the source code on your machine by-
+
+    `git clone https://github.com/sitture/techmiya-commercials.git`
+
+3. Create a python virtual environment and install python and django related dependencies.
+
+    ```shell
+    cd website
```
Comment (haroon-sheikh) [inline README.md]: should this be `cd techmiya-commercials`?

**Review by haroon-sheikh — CHANGES_REQUESTED** [review]

File: requirements/dev.txt (hunk 1 above)
Comment (HargovindArora) [reply requirements/dev.txt:1]: So which version do you want?

File: README.md (hunk 2 above)
Comment (HargovindArora) [reply README.md]: @haroon-sheikh Oh. Doing that

**Pull request merged**